### ISSUE TITLE
some characters in the title cause an error on Windows

### ISSUE THREAD

**Issue by agmmnn** [lovasoa/dezoomify-rs#29]
when try to download from _artsandculture.google.com_ if there are _some characters_ in the title like `"`, `'`, `?` it gives error.

**Example Urls:**
[for: "](https://artsandculture.google.com/asset/is-it-so-nominated-in-the-bond-scene-from-the-merchant-of-venice-john-rogers-american-1829-1904/aQHU6ANVrsf7Xw), [for: '](https://artsandculture.google.com/asset/the-rocky-mountains-lander-s-peak-albert-bierstadt/2gHyeS2wcvUJeA) [for: ?](https://artsandculture.google.com/asset/question-mark-australian-wwi-poster-norman-lindsay/qgEAkjrSADt2HA)

**Screenshot:**
for example: [The Rocky Mountains, Lander's Peak](https://artsandculture.google.com/asset/the-rocky-mountains-lander-s-peak-albert-bierstadt/2gHyeS2wcvUJeA)
![ApplicationFrameHost_xbejd3gEkx](https://user-images.githubusercontent.com/16024979/77046762-9e6abd00-69d4-11ea-945c-b8e7b8e80aa2.png)
_Lander's_, `'` character turns into utf16 `\u0027`. and so apparently these escapes don't work: `\'` and `\"`  (also i try [unicode escapes](https://doc.rust-lang.org/reference/tokens.html#unicode-escapes), `\u{0027}` and `\x27` it didnt work either) actually it works but as i understand the title is already converted to utf16 before coming to `name.replace` function. i want to fix that but rust seems complicated to me :)

my temporary solution adding `\\`. (_Lander\u0027s -> Landeru0027s_)
```rust
-format!("{}.jpg", name.replace(&['(', ')', ',', '\"', '.', ';', ':', '\''][..], ""))
+format!("{}.jpg", name.replace(&['?', '\\', '(', ')', ',', '\"', '.', ';', ':', '\''][..], ""))
```

__System:__
Windows 10 64bit (1909)(TR)

**Comment by lovasoa** [comment]
Hmm... This looks like a windows-specific problem. I'll look into that, thank you very much for the report !

**Comment by lovasoa** [comment]
Fixed in dezoomify-rs v1.8.4 :smiley:

**Comment by agmmnn** [comment]
thank you so much for your effort ^_^
